iosAppPreview/iosAppPreview/ContentView.swift

```diff
@@ -57,6 +59,7 @@ struct ContentView: View {
         @Published var isPresented: Bool = false
         var current: AssessmentState? {
             didSet {
+                print("---")
```

Comment:
Is this for debugging purposes/log file clarity? Or am I missing the point?

Reply:
Yes. It's a preview app and the "---" is so that for each run of the assessment, it's separated. This isn't in any app or code that is run other than during development